### ISSUE TITLE
[linux] the Installer installs files with unix permissions 777 RWX

### ISSUE THREAD

**Issue by dreamcat4** [sqrldev/SQRLDotNetClient#84]
This doesnt seem very secure, for a piece of security minded software. The containing `SQRL` folder (not shown here) is also 777 too.

```sh
$ ll
total 170M
drwxrwxrwx 3 id id 4.0K Apr  3 07:24 Assets/
-rwxrwxrwx 1 id id 300K Apr  3 07:24 AtkSharp.dll*
-rwxrwxrwx 1 id id  39K Apr  3 07:24 Avalonia.Animation.dll*
-rwxrwxrwx 1 id id 152K Apr  3 07:24 Avalonia.Base.dll*
-rwxrwxrwx 1 id id 520K Apr  3 07:24 Avalonia.Controls.dll*
-rwxrwxrwx 1 id id  25K Apr  3 07:24 Avalonia.DesignerSupport.dll*
-rwxrwxrwx 1 id id 5.0K Apr  3 07:24 Avalonia.Desktop.dll*
-rwxrwxrwx 1 id id  25K Apr  3 07:24 Avalonia.DesktopRuntime.dll*
-rwxrwxrwx 1 id id  58K Apr  3 07:24 Avalonia.Diagnostics.dll*
-rwxrwxrwx 1 id id 232K Apr  3 07:24 Avalonia.Dialogs.dll*
-rwxrwxrwx 1 id id  98K Apr  3 07:24 Avalonia.Direct2D1.dll*
-rwxrwxrwx 1 id id 3.5K Apr  3 07:24 Avalonia.dll*
-rwxrwxrwx 1 id id  33K Apr  3 07:24 Avalonia.FreeDesktop.dll*
-rwxrwxrwx 1 id id  72K Apr  3 07:24 Avalonia.Input.dll*
-rwxrwxrwx 1 id id  17K Apr  3 07:24 Avalonia.Interactivity.dll*
-rwxrwxrwx 1 id id  49K Apr  3 07:24 Avalonia.Layout.dll*
-rwxrwxrwx 1 id id 7.5K Apr  3 07:24 Avalonia.Logging.Serilog.dll*
-rwxrwxrwx 1 id id  38K Apr  3 07:24 Avalonia.Markup.dll*
-rwxrwxrwx 1 id id 251K Apr  3 07:24 Avalonia.Markup.Xaml.dll*
-rwxrwxrwx 1 id id  73K Apr  3 07:24 Avalonia.Native.dll*
-rwxrwxrwx 1 id id  61K Apr  3 07:24 Avalonia.OpenGL.dll*
-rwxrwxrwx 1 id id  19K Apr  3 07:24 Avalonia.ReactiveUI.dll*
-rwxrwxrwx 1 id id  55K Apr  3 07:24 Avalonia.Remote.Protocol.dll*
-rwxrwxrwx 1 id id  65K Apr  3 07:24 Avalonia.Skia.dll*
-rwxrwxrwx 1 id id  60K Apr  3 07:24 Avalonia.Styling.dll*
-rwxrwxrwx 1 id id 276K Apr  3 07:24 Avalonia.Themes.Default.dll*
-rwxrwxrwx 1 id id 201K Apr  3 07:24 Avalonia.Visuals.dll*
-rwxrwxrwx 1 id id  98K Apr  3 07:24 Avalonia.Win32.dll*
-rwxrwxrwx 1 id id 196K Apr  3 07:24 Avalonia.X11.dll*
-rwxrwxrwx 1 id id  12K Apr  3 07:24 Avalonia.Xaml.Interactions.Custom.dll*
-rwxrwxrwx 1 id id  22K Apr  3 07:24 Avalonia.Xaml.Interactions.dll*
-rwxrwxrwx 1 id id  13K Apr  3 07:24 Avalonia.Xaml.Interactivity.dll*
-rwxrwxrwx 1 id id 115K Apr  3 07:24 CairoSharp.dll*
-rwxrwxrwx 1 id id 108K Apr  3 07:24 createdump*
-rwxrwxrwx 1 id id 572K Apr  3 07:24 DynamicData.dll*
-rwxrwxrwx 1 id id 696K Apr  3 07:24 Eto.dll*
-rwxrwxrwx 1 id id 389K Apr  3 07:24 Eto.Gtk.dll*
-rwxrwxrwx 1 id id 604K Apr  3 07:24 GdkSharp.dll*
-rwxrwxrwx 1 id id 994K Apr  3 07:24 GioSharp.dll*
-rwxrwxrwx 1 id id  14K Apr  3 07:24 GitAPIHubHelper.dll*
-rwxrwxrwx 1 id id 4.9K Apr  3 07:24 GitAPIHubHelper.pdb*
-rwxrwxrwx 1 id id 250K Apr  3 07:24 GLibSharp.dll*
-rwxrwxrwx 1 id id 3.3M Apr  3 07:24 GtkSharp.dll*
-rwxrwxrwx 1 id id  61K Apr  3 07:24 JetBrains.Annotations.dll*
-rwxrwxrwx 1 id id 3.0M Apr  3 07:24 libclrjit.so*
-rwxrwxrwx 1 id id 9.0M Apr  3 07:24 libcoreclr.so*
-rwxrwxrwx 1 id id 723K Apr  3 07:24 libcoreclrtraceptprovider.so*
-rwxrwxrwx 1 id id 905K Apr  3 07:24 libdbgshim.so*
-rwxrwxrwx 1 id id 1.1M Apr  3 07:24 libe_sqlite3.so*
-rwxrwxrwx 1 id id 368K Apr  3 07:24 libhostfxr.so*
-rwxrwxrwx 1 id id 324K Apr  3 07:24 libhostpolicy.so*
-rwxrwxrwx 1 id id 3.4M Apr  3 07:24 libmscordaccore.so*
-rwxrwxrwx 1 id id 2.6M Apr  3 07:24 libmscordbi.so*
-rwxrwxrwx 1 id id 9.5M Apr  3 07:24 libSkiaSharp.so*
-rwxrwxrwx 1 id id 343K Apr  3 07:24 libsodium.so*
-rw-rw-r-- 1 id id  99K Apr  3 23:45 log20200403.txt
-rw-rw-r-- 1 id id  47K Apr  4 08:52 log20200404.txt
-rwxrwxrwx 1 id id 5.0K Apr  3 07:24 Microsoft.Bcl.AsyncInterfaces.dll*
-rwxrwxrwx 1 id id 4.5K Apr  3 07:24 Microsoft.Bcl.HashCode.dll*
-rwxrwxrwx 1 id id 4.0M Apr  3 07:24 Microsoft.CodeAnalysis.CSharp.dll*
-rwxrwxrwx 1 id id 2.0M Apr  3 07:24 Microsoft.CodeAnalysis.dll*
-rwxrwxrwx 1 id id 4.8M Apr  3 07:24 Microsoft.CodeAnalysis.VisualBasic.dll*
-rwxrwxrwx 1 id id 134K Apr  3 07:24 Microsoft.Data.Sqlite.dll*
-rwxrwxrwx 1 id id  14K Apr  3 07:24 Microsoft.DotNet.PlatformAbstractions.dll*
-rwxrwxrwx 1 id id  11K Apr  3 07:24 Microsoft.EntityFrameworkCore.Abstractions.dll*
-rwxrwxrwx 1 id id 1.5M Apr  3 07:24 Microsoft.EntityFrameworkCore.dll*
-rwxrwxrwx 1 id id 825K Apr  3 07:24 Microsoft.EntityFrameworkCore.Relational.dll*
-rwxrwxrwx 1 id id 134K Apr  3 07:24 Microsoft.EntityFrameworkCore.Sqlite.dll*
-rwxrwxrwx 1 id id  17K Apr  3 07:24 Microsoft.Extensions.Caching.Abstractions.dll*
-rwxrwxrwx 1 id id  23K Apr  3 07:24 Microsoft.Extensions.Caching.Memory.dll*
-rwxrwxrwx 1 id id  12K Apr  3 07:24 Microsoft.Extensions.Configuration.Abstractions.dll*
-rwxrwxrwx 1 id id  16K Apr  3 07:24 Microsoft.Extensions.Configuration.Binder.dll*
-rwxrwxrwx 1 id id  18K Apr  3 07:24 Microsoft.Extensions.Configuration.dll*
-rwxrwxrwx 1 id id  28K Apr  3 07:24 Microsoft.Extensions.DependencyInjection.Abstractions.dll*
-rwxrwxrwx 1 id id  61K Apr  3 07:24 Microsoft.Extensions.DependencyInjection.dll*
-rwxrwxrwx 1 id id  57K Apr  3 07:24 Microsoft.Extensions.DependencyModel.dll*
-rwxrwxrwx 1 id id  38K Apr  3 07:24 Microsoft.Extensions.Logging.Abstractions.dll*
-rwxrwxrwx 1 id id  24K Apr  3 07:24 Microsoft.Extensions.Logging.dll*
-rwxrwxrwx 1 id id  41K Apr  3 07:24 Microsoft.Extensions.Options.dll*
-rwxrwxrwx 1 id id  29K Apr  3 07:24 Microsoft.Extensions.Primitives.dll*
-rwxrwxrwx 1 id id 168K Apr  3 07:24 Microsoft.VisualBasic.dll*
-rwxrwxrwx 1 id id 6.5K Apr  3 07:24 Microsoft.Win32.Primitives.dll*
-rwxrwxrwx 1 id id  24K Apr  3 07:24 Microsoft.Win32.Registry.dll*
-rwxrwxrwx 1 id id  13K Apr  3 07:24 Microsoft.Win32.SystemEvents.dll*
-rwxrwxrwx 1 id id 5.1M Apr  3 07:24 MonoMac.dll*
-rwxrwxrwx 1 id id  56K Apr  3 07:24 mscorlib.dll*
-rwxrwxrwx 1 id id 667K Apr  3 07:24 Newtonsoft.Json.dll*
-rwxrwxrwx 1 id id 196K Apr  3 07:24 PangoSharp.dll*
-rwxrwxrwx 1 id id 224K Apr  3 07:24 protobuf-net.dll*
-rwxrwxrwx 1 id id 112K Apr  3 07:24 QRCoder.dll*
-rwxrwxrwx 1 id id 202K Apr  3 07:24 ReactiveUI.dll*
-rwxrwxrwx 1 id id 123K Apr  3 07:24 Serilog.dll*
-rwxrwxrwx 1 id id  32K Apr  3 07:24 Serilog.Sinks.Console.dll*
-rwxrwxrwx 1 id id 6.5K Apr  3 07:24 Serilog.Sinks.Debug.dll*
-rwxrwxrwx 1 id id  27K Apr  3 07:24 Serilog.Sinks.File.dll*
-rwxrwxrwx 1 id id 6.0K Apr  3 07:24 Serilog.Sinks.Trace.dll*
-rwxrwxrwx 1 id id 478K Apr  3 07:24 SharpDX.Direct2D1.dll*
-rwxrwxrwx 1 id id 272K Apr  3 07:24 SharpDX.Direct3D11.dll*
-rwxrwxrwx 1 id id 266K Apr  3 07:24 SharpDX.dll*
-rwxrwxrwx 1 id id 135K Apr  3 07:24 SharpDX.DXGI.dll*
-rwxrwxrwx 1 id id 160K Apr  3 07:24 SharpGen.Runtime.COM.dll*
-rwxrwxrwx 1 id id  30K Apr  3 07:24 SharpGen.Runtime.dll*
-rwxrwxrwx 1 id id 280K Apr  3 07:24 SkiaSharp.dll*
-rwxrwxrwx 1 id id  40K Apr  3 07:24 Sodium.Core.ForSqrl.dll*
-rwxrwxrwx 1 id id  272 Apr  3 07:24 SOS_README.md*
-rwxrwxrwx 1 id id  78K Apr  3 07:24 Splat.dll*
-rwxrwxrwx 1 id id 5.5K Apr  3 07:24 SQLitePCLRaw.batteries_v2.dll*
-rwxrwxrwx 1 id id  45K Apr  3 07:24 SQLitePCLRaw.core.dll*
-rwxrwxrwx 1 id id 5.0K Apr  3 07:24 SQLitePCLRaw.nativelibrary.dll*
-rwxrwxrwx 1 id id  55K Apr  3 07:24 SQLitePCLRaw.provider.dynamic_cdecl.dll*
-rwxrwxrwx 1 id id  28K Apr  3 07:27 sqrl.db*
-rwxrwxrwx 1 id id  194 Apr  3 07:24 sqrldev-sqrl.desktop*
-rwxrwxrwx 1 id id  85K Apr  3 07:24 SQRLDotNetClientUI*
-rwxrwxrwx 1 id id 185K Apr  3 07:24 SQRLDotNetClientUI.deps.json*
-rwxrwxrwx 1 id id 1.5M Apr  3 07:24 SQRLDotNetClientUI.dll*
-rwxrwxrwx 1 id id  51K Apr  3 07:24 SQRLDotNetClientUI.pdb*
-rwxrwxrwx 1 id id  185 Apr  3 07:24 SQRLDotNetClientUI.runtimeconfig.json*
-rwxrwxrwx 1 id id  75M Apr  3 07:18 SQRLPlatformAwareInstaller_linux*
-rwxrwxrwx 1 id id 3.0K Apr  3 07:24 SQRL.png*
-rwxrwxrwx 1 id id  54K Apr  3 07:24 SQRLUtilsLib.dll*
-rwxrwxrwx 1 id id  17K Apr  3 07:24 SQRLUtilsLib.pdb*
-rwxrwxrwx 1 id id   12 Apr  3 07:24 sqrlversion.json*
-rwxrwxrwx 1 id id 188K Apr  3 07:24 System.Collections.Concurrent.dll*
-rwxrwxrwx 1 id id 332K Apr  3 07:24 System.Collections.dll*
-rwxrwxrwx 1 id id 163K Apr  3 07:24 System.Collections.Immutable.dll*
-rwxrwxrwx 1 id id  34K Apr  3 07:24 System.Collections.NonGeneric.dll*
-rwxrwxrwx 1 id id  31K Apr  3 07:24 System.Collections.Specialized.dll*
-rwxrwxrwx 1 id id  68K Apr  3 07:24 System.ComponentModel.Annotations.dll*
-rwxrwxrwx 1 id id 5.0K Apr  3 07:24 System.ComponentModel.dll*
-rwxrwxrwx 1 id id  14K Apr  3 07:24 System.ComponentModel.EventBasedAsync.dll*
-rwxrwxrwx 1 id id  21K Apr  3 07:24 System.ComponentModel.Primitives.dll*
-rwxrwxrwx 1 id id 252K Apr  3 07:24 System.ComponentModel.TypeConverter.dll*
-rwxrwxrwx 1 id id  71K Apr  3 07:24 System.Console.dll*
-rwxrwxrwx 1 id id 927K Apr  3 07:24 System.Data.Common.dll*
-rwxrwxrwx 1 id id  40K Apr  3 07:24 System.Diagnostics.DiagnosticSource.dll*
-rwxrwxrwx 1 id id  91K Apr  3 07:24 System.Diagnostics.Process.dll*
-rwxrwxrwx 1 id id  36K Apr  3 07:24 System.Diagnostics.StackTrace.dll*
-rwxrwxrwx 1 id id  18K Apr  3 07:24 System.Diagnostics.Tools.dll*
-rwxrwxrwx 1 id id 124K Apr  3 07:24 System.Diagnostics.TraceSource.dll*
-rwxrwxrwx 1 id id 354K Apr  3 07:24 System.Drawing.Common.dll*
-rwxrwxrwx 1 id id  46K Apr  3 07:24 System.Drawing.Primitives.dll*
-rwxrwxrwx 1 id id  53K Apr  3 07:24 System.Globalization.Native.so*
-rwxrwxrwx 1 id id  27K Apr  3 07:24 System.IO.Compression.Brotli.dll*
-rwxrwxrwx 1 id id  90K Apr  3 07:24 System.IO.Compression.dll*
-rwxrwxrwx 1 id id 2.8M Apr  3 07:24 System.IO.Compression.Native.a*
-rwxrwxrwx 1 id id 771K Apr  3 07:24 System.IO.Compression.Native.so*
-rwxrwxrwx 1 id id  13K Apr  3 07:24 System.IO.FileSystem.AccessControl.dll*
-rwxrwxrwx 1 id id  77K Apr  3 07:24 System.IO.FileSystem.dll*
-rwxrwxrwx 1 id id  32K Apr  3 07:24 System.IO.FileSystem.DriveInfo.dll*
-rwxrwxrwx 1 id id  29K Apr  3 07:24 System.IO.IsolatedStorage.dll*
-rwxrwxrwx 1 id id  30K Apr  3 07:24 System.IO.MemoryMappedFiles.dll*
-rwxrwxrwx 1 id id 124K Apr  3 07:24 System.Linq.dll*
-rwxrwxrwx 1 id id 528K Apr  3 07:24 System.Linq.Expressions.dll*
-rwxrwxrwx 1 id id 205K Apr  3 07:24 System.Linq.Parallel.dll*
-rwxrwxrwx 1 id id  57K Apr  3 07:24 System.Linq.Queryable.dll*
-rwxrwxrwx 1 id id 185K Apr  3 07:24 System.Memory.dll*
-rwxrwxrwx 1 id id 411K Apr  3 07:24 System.Native.a*
-rwxrwxrwx 1 id id  64K Apr  3 07:24 System.Native.so*
-rwxrwxrwx 1 id id 524K Apr  3 07:24 System.Net.Http.dll*
-rwxrwxrwx 1 id id 117K Apr  3 07:24 System.Net.HttpListener.dll*
-rwxrwxrwx 1 id id  79K Apr  3 07:24 System.Net.Http.Native.a*
-rwxrwxrwx 1 id id  15K Apr  3 07:24 System.Net.Http.Native.so*
-rwxrwxrwx 1 id id  33K Apr  3 07:24 System.Net.NameResolution.dll*
-rwxrwxrwx 1 id id  73K Apr  3 07:24 System.Net.NetworkInformation.dll*
-rwxrwxrwx 1 id id  82K Apr  3 07:24 System.Net.Primitives.dll*
-rwxrwxrwx 1 id id 127K Apr  3 07:24 System.Net.Requests.dll*
-rwxrwxrwx 1 id id 235K Apr  3 07:24 System.Net.Security.dll*
-rwxrwxrwx 1 id id  36K Apr  3 07:24 System.Net.Security.Native.a*
-rwxrwxrwx 1 id id  15K Apr  3 07:24 System.Net.Security.Native.so*
-rwxrwxrwx 1 id id  14K Apr  3 07:24 System.Net.ServicePoint.dll*
-rwxrwxrwx 1 id id 203K Apr  3 07:24 System.Net.Sockets.dll*
-rwxrwxrwx 1 id id  56K Apr  3 07:24 System.Net.WebClient.dll*
-rwxrwxrwx 1 id id  25K Apr  3 07:24 System.Net.WebHeaderCollection.dll*
-rwxrwxrwx 1 id id 9.0K Apr  3 07:24 System.Net.WebProxy.dll*
-rwxrwxrwx 1 id id  30K Apr  3 07:24 System.Net.WebSockets.Client.dll*
-rwxrwxrwx 1 id id  49K Apr  3 07:24 System.Net.WebSockets.dll*
-rwxrwxrwx 1 id id  34K Apr  3 07:24 System.ObjectModel.dll*
-rwxrwxrwx 1 id id 8.8M Apr  3 07:24 System.Private.CoreLib.dll*
-rwxrwxrwx 1 id id 735K Apr  3 07:24 System.Private.DataContractSerialization.dll*
-rwxrwxrwx 1 id id 1.8M Apr  3 07:24 System.Private.ServiceModel.dll*
-rwxrwxrwx 1 id id 240K Apr  3 07:24 System.Private.Uri.dll*
-rwxrwxrwx 1 id id 3.0M Apr  3 07:24 System.Private.Xml.dll*
-rwxrwxrwx 1 id id 130K Apr  3 07:24 System.Private.Xml.Linq.dll*
-rwxrwxrwx 1 id id 1.1M Apr  3 07:24 System.Reactive.dll*
-rwxrwxrwx 1 id id  24K Apr  3 07:24 System.Reflection.DispatchProxy.dll*
-rwxrwxrwx 1 id id 424K Apr  3 07:24 System.Reflection.Metadata.dll*
-rwxrwxrwx 1 id id  30K Apr  3 07:24 System.Reflection.TypeExtensions.dll*
-rwxrwxrwx 1 id id  15K Apr  3 07:24 System.Resources.Writer.dll*
-rwxrwxrwx 1 id id 5.0K Apr  3 07:24 System.Runtime.CompilerServices.Unsafe.dll*
-rwxrwxrwx 1 id id 6.0K Apr  3 07:24 System.Runtime.CompilerServices.VisualC.dll*
-rwxrwxrwx 1 id id  53K Apr  3 07:24 System.Runtime.dll*
-rwxrwxrwx 1 id id 205K Apr  3 07:24 System.Runtime.Extensions.dll*
-rwxrwxrwx 1 id id  53K Apr  3 07:24 System.Runtime.InteropServices.dll*
-rwxrwxrwx 1 id id 9.0K Apr  3 07:24 System.Runtime.InteropServices.RuntimeInformation.dll*
-rwxrwxrwx 1 id id 7.5K Apr  3 07:24 System.Runtime.InteropServices.WindowsRuntime.dll*
-rwxrwxrwx 1 id id  61K Apr  3 07:24 System.Runtime.Numerics.dll*
-rwxrwxrwx 1 id id 306K Apr  3 07:24 System.Runtime.Serialization.Formatters.dll*
-rwxrwxrwx 1 id id  10K Apr  3 07:24 System.Runtime.Serialization.Primitives.dll*
-rwxrwxrwx 1 id id  30K Apr  3 07:24 System.Security.AccessControl.dll*
-rwxrwxrwx 1 id id  38K Apr  3 07:24 System.Security.Claims.dll*
-rwxrwxrwx 1 id id 204K Apr  3 07:24 System.Security.Cryptography.Algorithms.dll*
-rwxrwxrwx 1 id id  40K Apr  3 07:24 System.Security.Cryptography.Csp.dll*
-rwxrwxrwx 1 id id  31K Apr  3 07:24 System.Security.Cryptography.Encoding.dll*
-rwxrwxrwx 1 id id 970K Apr  3 07:24 System.Security.Cryptography.Native.OpenSsl.a*
-rwxrwxrwx 1 id id 115K Apr  3 07:24 System.Security.Cryptography.Native.OpenSsl.so*
-rwxrwxrwx 1 id id  65K Apr  3 07:24 System.Security.Cryptography.OpenSsl.dll*
-rwxrwxrwx 1 id id  40K Apr  3 07:24 System.Security.Cryptography.Primitives.dll*
-rwxrwxrwx 1 id id 213K Apr  3 07:24 System.Security.Cryptography.X509Certificates.dll*
-rwxrwxrwx 1 id id  16K Apr  3 07:24 System.Security.Principal.dll*
-rwxrwxrwx 1 id id  19K Apr  3 07:24 System.Security.Principal.Windows.dll*
-rwxrwxrwx 1 id id  10K Apr  3 07:24 System.ServiceModel.dll*
-rwxrwxrwx 1 id id  12K Apr  3 07:24 System.ServiceModel.Primitives.dll*
-rwxrwxrwx 1 id id  44K Apr  3 07:24 System.Text.Encodings.Web.dll*
-rwxrwxrwx 1 id id 278K Apr  3 07:24 System.Text.Json.dll*
-rwxrwxrwx 1 id id 133K Apr  3 07:24 System.Text.RegularExpressions.dll*
-rwxrwxrwx 1 id id  77K Apr  3 07:24 System.Threading.dll*
-rwxrwxrwx 1 id id  39K Apr  3 07:24 System.Threading.Tasks.Parallel.dll*
-rwxrwxrwx 1 id id 138K Apr  3 07:24 System.Transactions.Local.dll*
-rwxrwxrwx 1 id id  14K Apr  3 07:24 System.Web.HttpUtility.dll*
-rwxrwxrwx 1 id id 187K Apr  3 07:24 Tmds.DBus.dll*
-rwxrwxrwx 1 id id  40K Apr  3 07:24 X11.dll*
```

**Comment by dreamcat4** [comment]
Shouldn't that be `755` instead?

**Comment by alexhauser** [comment]
Thanks for the report, @dreamcat4. I've seen @josegomez change some permissions with one of the latest updates and meant to ask him about that, but fogot. So thanks for pointing it out. I'm sure @josegomez will update us on this soon.

**Comment by josegomez** [comment]
Yup had some issues with the SQL lite Db and changed some permissions to fix it 
We do need to clean that up the issue arose from requiring root/sudo to install but I was able to remove that too so we should be able to fix that

**Comment by josegomez** [comment]
Ugh see what I mean 
![image](https://user-images.githubusercontent.com/420837/78451843-dc631480-7655-11ea-879f-924edfe7b3f7.png)

Linux is gonna be the death of me! It's having permissions issue if when we don't use use sudo, but then that means all the files are creates as root which means running SQRL would require root (which we don't want)
So I change the permissions to fix that, I'll work on it some more.

**Comment by josegomez** [comment]
Working on this.

**Comment by josegomez** [comment]
Fixed on #94
